DataGrid Grid Summaries demo: rewrite and rename to "Total Summaries" (#616)
DevExpress/devextreme-demos: JSDemos/Demos/DataGrid/GridSummaries/description.md

> A total summary creates a synopsis for all data of a certain column. To configure this feature, add the [summary](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/) object, instatiate the **summary**.[totalItems](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/totalItems/) array and add summary objects. Each object must consist of at least the [column](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/totalItems/#column) and [summaryType](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/totalItems/#summaryType) properties. The former specifies the column that supplies data, the latter sets the aggregate function that applies to this data.

Reviewer: > To configure this feature, add the [summary](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/) object, instatiate the **summary**.[totalItems](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/totalItems/) array and add summary objects.

It's a difficult sentence to wrap one's head around because it gives instructions about code but the code itself is out there somewhere.

> instatiate 

means "to create an instance of something", which isn't what we do with the `totalItems` array.

> Each object must consist of

AFAIK, you can skip the `column` if you want a full data object in the `calculateCustomSummary` function.

```suggestion
A total summary creates a synopsis for all data of a certain column. To configure this feature, populate the [summary](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/).[totalItems](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/totalItems/) array with summary configuration objects. Each object should specify a [column](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/totalItems/#column) that supplies data for summary calculation and a [summaryType](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/totalItems/#summaryType)&mdash;an aggregate function that should be applied to this data.
```

> This demo has summaries that do the following:

Reviewer: "has" is a weak verb

```suggestion
This demo illustrates the following total summaries:
```

> 1. Count the number of orders (summaryType of *"count"*)

Reviewer: - Ordered list is good for series of connected steps. In our case, list items are independent of each other.
- If you can, describe all in one place, don't refer back to add to the previous description, as you do with `customizeText` and `valueFormat`.

```suggestion
- **Total number of orders**             
This summary uses the *"count"* **summaryType**.
```

> 2. Determine the date of the first order (summaryType of *"min"*)

Reviewer: ```suggestion
- **Date of the first order**              
This summary uses the *"min"* **summaryType** and the [customizeText](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/totalItems/#customizeText) function that formats the displayed summary value.
```

> 3. Calculate the total revenue (summaryType of *"sum"*).

Reviewer: ```suggestion
- **Total revenue**        
This summary uses the *"sum"* **summaryType** and the [valueFormat](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/totalItems/#valueFormat) property to format the summary value as currency.
```

> The second and the third summaries use the [customizeText](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/totalItems/#customizeText) and [valueFormat](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/totalItems/#valueFormat) properties to customize the displayed text and set the display format, respectively.

Reviewer: Since we gave this information in the list, this paragraph is no longer needed.

```suggestion
```

> A total summary creates a synopsis for all data of a certain column. To configure this feature, populate the [summary](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/).[totalItems](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/totalItems/) array with summary configuration objects. Each object should specify a [column](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/totalItems/#column) that supplies data for summary calculation and a [summaryType](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/totalItems/#summaryType)&mdash;an aggregate function that should be applied to this data.

Reviewer 2: I probably wouldn't use the word "synopsis" to explain "total summaries". The phrase "total summaries" seems self-explanatory and definitely closer to the real meaning than "synopsis". So my suggestion is to simply go directly into the second sentence: 

 ```suggestion
To display total summaries, populate the [summary](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/).[totalItems](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/totalItems/) array with configuration objects. Each object should specify a [column](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/totalItems/#column) that supplies data for summary calculation and a [summaryType](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/totalItems/#summaryType)&mdash;an aggregate function that should be applied to this data.
```
